concent_api/core/payments/backends/mock.py

```diff
@@ -20,8 +20,18 @@ def make_force_payment_to_provider(
     provider_eth_address: str,  # pylint: disable=unused-argument
     value: int,  # pylint: disable=unused-argument
     payment_ts: int,  # pylint: disable=unused-argument
-) -> None:
-    pass
+) -> Transaction:
+    return Transaction(
```

Comment:
BTW: maybe `MOCK_TRANSACTION` can be reused here?

Reply:
:+1: done